src/main/java/org/troyargonauts/subsystems/DriveTrain.java

```diff
@@ -102,15 +103,15 @@ public void tankDrive(double left, double right, double nerf) {
      * @return encoder position based on encoder values
      */
     public double getPosition() {
-        return (frontRight.getEncoder().getPosition() + frontLeft.getEncoder().getPosition()) / (2 * DriveConstants.GEARBOX_SCALE);
+        return (frontRight.getEncoder().getPosition() + frontLeft.getEncoder().getPosition()) / (2 * Constants.DriveTrain.GEARBOX_SCALE);
     }
 
     public double getLeftPosition() {
-        return frontLeft.getEncoder().getPosition() / DriveConstants.GEARBOX_SCALE;
+        return frontLeft.getEncoder().getPosition() / Constants.DriveTrain.GEARBOX_SCALE;
     }
 
     public double getRightPosition() {
-        return frontRight.getEncoder().getPosition() / DriveConstants.GEARBOX_SCALE;
+        return frontRight.getEncoder().getPosition() / Constants.DriveTrain.GEARBOX_SCALE;
     }
 
     /**
```

Comment:
"Resets the all DriveTrain encoders to a position of 0." Remove return tag

Reply:
This function does not reset the drivetrain encoders.

Remove gearbox scale.

Reply:
Unsure how that happened, oops. That is referring to the reset encoder function that I was unable to comment on I believe.